src/Data/Array/NonEmpty/Internal.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-exports.fold1Impl = function (f) {
+exports.foldr1Impl = function (f) {
```

Comment:
Isn't this the implementation for `foldl1`?

Reply:
yes, I changed a name, because this name is better

Reply:
Let me clarify. The function name is `foldr1Impl`, but the implementation is what I would define for `foldl1Impl`. My issue isn't that the you changed the name. My issue is that the name refers to the wrong implementation. If you changed this to `foldl1Impl` and changed the currently named `foldl1Impl` to `foldr1Impl`, then the names would refer to the correct functions. You would also need to update the tests, since they are referring to the wrong outputs as well.

Reply:
yes, fixed